lib/appbundler/app.rb

```diff
@@ -114,7 +114,11 @@ def file_format_comment
     # APPBUNDLER_ALLOW_RVM environment variable to "true". This feature
     # exists to make tests run correctly on travis.ci (which uses rvm).
     def env_sanitizer
-      %Q{ENV["GEM_HOME"] = ENV["GEM_PATH"] = nil unless ENV["APPBUNDLER_ALLOW_RVM"] == "true"}
+      <<-EOS
+ENV["GEM_HOME"] = ENV["GEM_PATH"] = nil unless ENV["APPBUNDLER_ALLOW_RVM"] == "true"
+require "rubygems"
+::Gem.clear_paths if ::Gem.respond_to?(:clear_paths)
```

Comment:
i don't think we care about anything old enough to need to bother with either requiring rubygems or protecting the Gem.clear_paths call.